core/src/main/java/uk/ac/bbsrc/tgac/miso/core/manager/ERASubmissionManager.java

```diff
@@ -415,6 +415,9 @@ protected void populate(Element xml, Study s) {
       centerProjectName.setTextContent(s.getProject().getAlias());
       studyDescriptor.appendChild(centerProjectName);
 
+      Element studyAbstract = xml.getOwnerDocument().createElementNS(null, "STUDY_ABSTRACT");
+      studyDescriptor.appendChild(studyAbstract);
```

Comment:
Yes. I'm not sure if this element is optional in the ERA schema, so I'd rather it be blank, which is what it was before.